src/style/app.module.css

```diff
@@ -525,13 +537,13 @@
   color: var(--black-color) !important;
   /* margin-top: 10px; */
   /* margin-left: 5px; */
-  border: 1px solid var(--brown-color);
+  border: 1px solid var(--dark-grey-border);
 }
 
 .createButton:hover {
   background-color: var(--grey-bg-color) !important;
   color: var(--black-color) !important;
-  border: 1px solid var(--brown-color) !important;
+  border: 1px solid var(--dark-grey-border) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`